### PR TITLE
Punj

### PR DESCRIPTION
**Pull request by aakash049**

### CODE REVIEW

File: Chinese/wow/wow.md [hunk 1]
```diff
@@ -0,0 +1 @@
+this is cool
```
Comment (casey-collab) [inline Chinese/wow/wow.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by aakash049** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: Chinese/wow/wow.md (hunk 1 above)
Comment (aakash049) [inline Chinese/wow/wow.md]: Please review this, @casey-collab Thanks Alot, you are awesome

**Comment by aakash049** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :cake:

**Pull request merged**